app/resources/api/v2/plate_purpose_resource.rb

```diff
@@ -13,34 +13,34 @@ class PlatePurposeResource < BaseResource
 
       # The following attributes are sent by Limber for a new plate purpose.
 
-      # @!attribute name
-      #  @return [String] gets or sets the name of the plate purpose
+      # @!attribute [rw]
```

Comment:
just curious, what does that exclamation mark do?
something to do with documentation markup?

Reply:
Yes, it's part of the way you write markup for the YARD documentation generator I will be relying on later to generate documentaton of the v2 API (different story in the To Do column) https://rubydoc.info/gems/yard/file/docs/Tags.md#attribute